src/domain/api/constants/ApiConstants.ts

```diff
@@ -0,0 +1,2 @@
+export const TUNNISTAMO_ENDPOINT = 'http://localhost:8000';
```

Comment:
Shouldn't this be in an .env-file? I guess this is still wip :-)

Reply:
The readme uses `tunnistamo-local` as hostname

Reply:
Yeah i will use env value